### ISSUE TITLE
Move accessory to another WiFi network?

### ISSUE THREAD

**Issue by olavt** [home-assistant-libs/python-matter-server#678]
People need to re-configure their WiFi networks from time to time. Having to rebuild the Matter fabrics / networks as a consequence would be very painful.

I'm not sure how moving a Matter accessory device from one WiFi network to another would be done using the Matter SDK, but it looks like Apple Home makes this quite easy.

This should maybe be a function in the Python Matter Server too?

**Comment by marcelveldt** [comment]
Yes we should create functionality for this. I need to dive into the details how to do this and if all devices support it.

**Comment by Apollon77** [comment]
I would think. the following needs to happen:
* establish a CASE session
* arm failsafe timer
* use NetworkCommissioning cluster to update and then connect to the new network
* discover and re-establish a/the new CASE session
* invoke commissioningComplete to "persist" the data
